Check broken links (#4)
christinerose/ocaml.org: data/changelog/odoc/2019-03-08-odoc-1.4.0.md

  - Paragraph headings (`{4 ...}`) and subparagraph headings (`{5 ...}`) (#217,
    Rizo Isrof).
  - `odoc support-files-targets` command (#232).

>   - Recommend [`bsdoc`](https://ostera.github.io/bsdoc/docs/BsDoc/ - [404 Not Found]) for using

Reviewer: ```suggestion
  - Recommend `bsdoc` for using
```
No longer in the user's repo

We are pleased to announce release **1.4.0** of [**odoc**](https://github.com/ocaml/odoc), the new, Dune-friendly documentation generator for OCaml and ReasonML.

> Some sample output, which you can browse [here](http://odis.io/odoc-demo/order/Order/index.html - [404 Not Found]):

Reviewer: ```suggestion
Some sample output, which you can browse [here](https://ocaml.github.io/odoc/):
```

> In summary, compared with 1.2.0, odoc now has:

Reviewer: ```suggestion
In summary, compared with 1.2.0, `odoc` now has:
```

> - [New appearance](http://odis.io/odoc-demo/order/Order/index.html - [404 Not Found]).

Reviewer: ```suggestion
- [New appearance](https://ocaml.github.io/odoc/).
```

- Tables of contents on each page.
- [Generation of docs in ReasonML syntax](https://github.com/ocaml/odoc/pull/156).

> - [Interop with BuckleScript](https://ostera.github.io/bsdoc/docs/BsDoc/ - [404 Not Found]).

Reviewer: ```suggestion
- [Interop with BuckleScript](https://discuss.ocaml.org/t/is-it-practical-to-mix-bucklescript-and-js-of-ocaml-in-a-single-project/5181).
```
Only thing I could find with Interop with BuckleScript for OCaml
